front/components/poke/data_sources/view.tsx

```diff
@@ -56,6 +63,12 @@ export function ViewDataSourceTable({
                 🤓 Show raw objects
               </PokeButton>
             </div>
+            {isPaused && isRunning && (
```

Comment:
Can we instead do two separate lines and possibly a warning if something looks out of order?

Reply:
Sorry, I'm confused by your message since your are pointing to a warning.

Reply:
Sorry I meant to keep both a cell for temporal workflow running and paused vs having just the warning and running

Reply:
I kept the cellls for paused & running. (pausedAt is a cell just below running).

The only things I did is use the temporal worklows for running state and I added a warning box if running + paused.

Reply:
commented below where I see the redline :+1: